# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/03 09:10:09+01:00 perex@suse.cz 
#   [ALSA] ifdef typos: sound_isa_es18xx.c
#   
#   ES18xx driver
#   Changed CONFIG_PNP_ to CONFIG_PNP, also fixed a comment related to
#   another CONFIG_PNP.
#   
#   Signed-off-by: Domen Puncer <domen@coderock.org>
#   Signed-off-by: Takashi Iwai <tiwai@suse.de>
# 
# sound/isa/es18xx.c
#   2004/11/30 07:34:07+01:00 perex@suse.cz +2 -2
#   [ALSA] ifdef typos: sound_isa_es18xx.c
#   
#   D:2004/11/30 14:34:07
#   C:ES18xx driver
#   F:isa/es18xx.c:1.51->1.52 
#   L:Changed CONFIG_PNP_ to CONFIG_PNP, also fixed a comment related to
#   L:another CONFIG_PNP.
#   Signed-off-by: Domen Puncer <domen@coderock.org>
#   Signed-off-by: Takashi Iwai <tiwai@suse.de>
# 
diff -Nru a/sound/isa/es18xx.c b/sound/isa/es18xx.c
--- a/sound/isa/es18xx.c	2005-01-22 23:12:29 -08:00
+++ b/sound/isa/es18xx.c	2005-01-22 23:12:29 -08:00
@@ -1849,7 +1849,7 @@
 static int isapnp[SNDRV_CARDS] = {[0 ... (SNDRV_CARDS - 1)] = 1};
 #endif
 static long port[SNDRV_CARDS] = SNDRV_DEFAULT_PORT;	/* 0x220,0x240,0x260,0x280 */
-#ifndef CONFIG_PNP_
+#ifndef CONFIG_PNP
 static long mpu_port[SNDRV_CARDS] = {[0 ... (SNDRV_CARDS - 1)] = -1};
 #else
 static long mpu_port[SNDRV_CARDS] = SNDRV_DEFAULT_PORT;
@@ -1988,7 +1988,7 @@
 	kfree(cfg);
 	return 0;
 }
-#endif /* CONFIG_PNP_ */
+#endif /* CONFIG_PNP */
 
 static int __devinit snd_audiodrive_probe(int dev, struct pnp_card_link *pcard,
 					  const struct pnp_card_device_id *pid)
